README_cn.md

```diff
@@ -55,6 +55,7 @@
 | [211-speech-to-text](notebooks/211-speech-to-text/)<br>[![Binder](https://mybinder.org/badge_logo.svg)](https://mybinder.org/v2/gh/openvinotoolkit/openvino_notebooks/HEAD?filepath=notebooks%2F211-speech-to-text%2F211-speech-to-text.ipynb) | 声音到文字的识别模型推理 | <img src="https://user-images.githubusercontent.com/36741649/140987347-279de058-55d7-4772-b013-0f2b12deaa61.png" width=225>|
 | [212-onnx-style-transfer](notebooks/212-onnx-style-transfer/)<br>[![Binder](https://mybinder.org/badge_logo.svg)](https://mybinder.org/v2/gh/openvinotoolkit/openvino_notebooks/HEAD?filepath=notebooks%2F212-onnx-style-transfer%2F212-onnx-style-transfer.ipynb) | 通过神经风格转换将图像转换为五种不同的风格 | <img src="https://user-images.githubusercontent.com/77325899/147358090-ff5b21f5-0efb-4aff-8444-9d07add49b92.png" width=100>→<img src="https://user-images.githubusercontent.com/77325899/147358009-0cf10d51-3150-40cb-a776-074558b98da5.png" width=100>|
 | [213-question-answering](notebooks/213-question-answering/)<br>[![Binder](https://mybinder.org/badge_logo.svg)](https://mybinder.org/v2/gh/openvinotoolkit/openvino_notebooks/HEAD?filepath=notebooks%2F213-question-answering%2F213-question-answering.ipynb) | 根据语境回答问题 | <img src="https://user-images.githubusercontent.com/4547501/152571639-ace628b2-e3d2-433e-8c28-9a5546d76a86.gif" width=225> |
+| [218-vehicle-detection-and-recognition](notebooks/218-vehicle-detection-and-recognition/)<br> | 车辆检测与属性识别 | <img src = "https://user-images.githubusercontent.com/47499836/157867020-99738b30-62ca-44e2-8d9e-caf13fb724ed.png" width=225> |
```

Comment:
```suggestion
| [218-vehicle-detection-and-recognition](notebooks/218-vehicle-detection-and-recognition/)<br> | 车辆检测与识别 | <img src = "https://user-images.githubusercontent.com/47499836/157867020-99738b30-62ca-44e2-8d9e-caf13fb724ed.png" width=225> |
```

Reply:
@IRONICBo In order to centralize data, please use data (images/video) that you could find here. You don't need to download that data, just use the URL instead of the data folder.

https://storage.openvinotoolkit.org/data/test_data/images/person-bicycle-car-detection.bmp
https://storage.openvinotoolkit.org/data/test_data/videos/car-detection.mp4
https://storage.openvinotoolkit.org/data/test_data/videos/person-bicycle-car-detection.mp4 

Use:
`!curl -o person-vehicle-bike.mp4 https://storage.openvinotoolkit.org/data/test_data/videos/person-bicycle-car-detection.mp4
`
@adrianboguszewski this is applicable for other PRs.

Reply:
@paularamo Thank you for your suggestion. I use the githubusercontent URL  instead of  the data folder.

Reply:
@IRONICBo Thanks. Yes, Please change the image to one of the references, and avoid putting the data folder, in order to use less space per notebook and avoid intellectual property issues on new images we include.

Reply:
@paularamo Ok, I have changed the image to https://storage.openvinotoolkit.org/data/test_data/images/person-bicycle-car-detection.bmp and remove the data folder.

Besides, I'll try to do model inference on video like https://storage.openvinotoolkit.org/data/test_data/videos/car-detection.mp4.

Reply:
@paularamo Thank you for your advice. I read the notebook of https://github.com/openvinotoolkit/openvino_notebooks/blob/main/notebooks/utils/notebook_utils.ipynb#L208 and I changed the "curl" with "download_file" function, And I it works well.